app/components/EmissionInfoButton/EmissionInfoButton.tsx

```diff
@@ -0,0 +1,33 @@
+import React from "react";
+import { useSelector } from "react-redux";
+import { pathOr } from "ramda";
+import { Ionicons } from "@expo/vector-icons";
+import { useNavigation, useRoute } from "@react-navigation/native";
+
+import { emissions } from "../../ducks";
+import { navigate } from "../../navigation";
+import styles from "./EmissionInfoButton.styles";
+
+const EmissionInfoButton = () => {
+  const navigation = useNavigation();
+  const navigator = navigate(navigation);
+
+  const route = useRoute();
+  const emissionId = pathOr("", ["params", "id"], route);
+
+  const { emissionType } =
+    useSelector((state) =>
+      emissions.selectors.getEmissionById(state, emissionId)
+    ) || {};
```

Comment:
no need for ` || {}` needed I think

Reply:
Since this is reused for the AddEmission page, the selector comes back as null on that page and then the destructure throws an exception

Reply:
ah yes you right :)